### DISCUSSION TITLE
Is image supported as an Action output?

### DISCUSSION THREAD

**Discussion by mhaowork** [connery-io/connery-sdk#117] [Q&A] [answered]
Hi, I am trying to support images for my local GPT assistant. Will Connery Action be able to support outputting images? It doesn't appear so if I am reading `plugin-definition-validation-utils.ts` correctly. Any suggestions?

Thanks!

**Comment by machulav** [comment, marked as answer]
Hi @mhaowork, you're right - currently, only string outputs are supported. As a workaround, you may convert the image to base64 and return it as a string. Then, on a client, convert it back to the image. Would it work in your case?

**Comment by mhaowork** [comment]
Thanks for the reply! Sounds good!
